manifests/init.pp

```diff
@@ -193,7 +193,7 @@
     include postfix::mta
   }
 
-  if $satellite {
+  if ( $satellite ) or ( $relayhost ) {
```

Comment:
is that logic correcr? Should t it be `and` and not `or`?

Reply:
@bastelfreak `or` is correct here because the goal is to effectively set `$satellite = true` when `$relayhost` is defined, without having to explicitly set `$satellite = true`.